cmd/kava/opendb/opendb_rocksdb.go

```diff
@@ -168,3 +197,42 @@ func newDefaultOptions() *grocksdb.Options {
 
 	return opts
 }
+
+// reportMetrics periodically requests stats from rocksdb and reports to prometheus
+// NOTE: should be launched as a goroutine
+func reportMetrics(db *grocksdb.DB, interval time.Duration) {
+	ticker := time.NewTicker(interval)
+	for {
+		select {
+		case <-ticker.C:
+			props, stats, err := getPropsAndStats(db)
+			if err != nil {
+				continue
```

Comment:
nit: no logging in any of these changes, I know this code is simple enough that it will probably "just work", but I think it's a good habit to add logging by default so there is more information for future developers for when things go wrong / need to be debugged (e.g. having a log statement on whether the metrics are enabled or not to allow the debugger to rule out issues related to setting up the metric routine, or here in this line to capture the error so it can be used to resolve the problem - versus throwing it away and losing information)

Reply:
@galxy25 
yeah, make sense
could you pls. advise what library to use, and point to an example in kava repo?
I want to make sure logging is consistent across kava repo